app/models/manageiq/providers/vmware/infra_manager/provision/cloning.rb

```diff
@@ -35,6 +35,7 @@ def prepare_for_clone_task
       :datastore     => dest_datastore,
       :folder        => dest_folder,
       :pool          => dest_resource_pool,
+      :profile       => dest_storage_profile,
```

Comment:
Thoughts on changing this option (and the option in the `vim_clone_options`) to `:storage_profile` instead of just `:profile`?